docs/inputs.md

```diff
@@ -25,54 +25,26 @@ The table below contains brief descriptions of the input files required to execu
 | special_fares.txt | Fares to coaster | Text File | Transportation Modelers | 
-| **Network: Active Transportation** |  |  |  | 
+| [mobilityHubMGRA.csv](#mobility_mgra) |  | CSV | Transportation Modelers | 
 | [SANDAG_Bike_Net.dbf](#bike_net_link) | Bike network links | DBF | GIS | 
 | [SANDAG_Bike_Node.dbf](#bike_net_node) | Bike network nodes | DBF | GIS | 
 | [bikeTazLogsum.csv](#bike_taz_logsum) <i>(not saved in inputs, instead, run at the beginning of a model run)<i> | Bike TAZ logsum | CSV | Transportation Modelers | 
 | [bikeMgraLogsum.csv](#bike_mgra_logsum) <i>(not saved in inputs, instead, run at the beginning of a model run)<i> | Bike MGRA logsum | CSV | Transportation Modelers | 
-| [walkMgraEquivMinutes.csv](#walk_mgra_equiv) <i>(not saved in inputs, instead, run at the beginning of a model run)<i> | Walk, in minutes, between MGRAs | CSV |  | |  |  |  |
-| **Visitor Model (Derived from visitor survey)** |  |  |  | 
-| visitor_businessFrequency.csv | Visitor model tour frequency distribution for business travelers | CSV | Transportation Modelers | 
-| visitor_personalFrequency.csv | Visitor model tour frequency distribution for personal travelers | CSV | Transportation Modelers | 
-| visitor_partySize.csv | Visitor model party size distribution | CSV | Transportation Modelers | 
-| visitor_autoAvailable.csv | Visitor model auto availability distribution | CSV | Transportation Modelers | 
-| visitor_income.csv | Visitor model income distribution | CSV | Transportation Modelers | 
-| visitor_tourTOD.csv | Visitor model tour time-of-day distribution | CSV | Transportation Modelers | 
-| visitor_stopFrequency.csv | Visitor model stop frequency distribution | CSV | Transportation Modelers | 
-| visitor_stopPurpose.csv | Visitor model stop purpose distribution | CSV | Transportation Modelers | 
-| visitor_outboundStopDuration.csv | Visitor model time-of-day offsets for outbound stops | CSV | Transportation Modelers | 
-| visitor_inboundStopDuration.csv | Visitor model time-of-day offsets for inbound stops | CSV | Transportation Modelers | 
-| **Airport Model (Derived from airport survey)** |  |  |  | 
-| [airport_purpose.csv](#airport_trip_purpose) | Airport model tour purpose frequency table | CSV | Transportation Modelers | 
-| [airport_party.csv](#airport_party_purpose) | Airport model party type frequency table | CSV | Transportation Modelers | 
-| [airport_nights.csv](#airport_nights) | Airport model trip duration frequency table | CSV | Transportation Modelers | 
-| [airport_income.csv](#airport_income) | Airport model trip income distribution table | CSV | Transportation Modelers | 
-| [airport_departure.csv](#airport_departure) | Airport model time-of-day distribution for departing trips | CSV | Transportation Modelers | 
-| [airport_arrival.csv](#airport_arrival) | Airport model time-of-day distribution for arriving trips | CSV | Transportation Modelers | 
+| mgra15.sbn | | SBN | |
```

Comment:
This should be shapefile type "mgra15.shp" @mansisandag

Reply:
Creating another PR because of branch issues to address this.